components/editor/vim/Makefile

```diff
@@ -79,6 +80,7 @@ $(X_VARIANT)/.configured:	CONFIGURE_OPTIONS += --with-vim-name=gvim
 $(X_VARIANT)/.configured:	CONFIGURE_OPTIONS += --with-x
 $(X_VARIANT)/.configured:	CONFIGURE_OPTIONS += --enable-gui=gtk2
 $(X_VARIANT)/.configured:	CONFIGURE_OPTIONS += --enable-gtk2-check
+$(X_VARIANT)/.configured:       CONFIGURE_OPTIONS += --with-terminal 
```

Comment:
Is something wrong with spaces?

Reply:
Should it be --enable-terminal ? Also, why this option is enabled only for X_VARIANT?

Reply:
@kenmays